strategy/to_json/from_serializable.ts

```diff
@@ -10,6 +10,10 @@ export function fromSerializable(): ToJSONStrategy {
   ): JSONValue => {
     if (Array.isArray(value)) {
       return value.map((item) => item.tsSerialize());
+    } else if (value === null) {
+      // Objects can be null, return them without further serialization.
+      // Double cast required to return a JSONValue
+      return value as null as JSONValue;
```

Comment:
interesting we need the double cast when the compiler can infer the first cast from the `if`

Reply:
Yea, what happens is the linter has an infinite recursion error occur when I view `JSONObject` in VSCode, unsure if it's related.

I did not find a cleaner way to do this yet.

Reply:
interesting error then!